doc/lsp.txt

```diff
@@ -306,6 +300,30 @@ completionMatcher	|String| option.  Enable fuzzy or case insensitive
 			    case  - case sensitive matching (default).
 			    fuzzy - fuzzy match completion items.
 			    icase - ignore case when matching items.
+diagLineHL		|String| option. The highlight group used for a line
+			with one or more diagnostics.  By default uses
+			"DiffAdd".  Use "NONE" to disable.
+diagVirtualTextHL	|String| option.  The highlight group used for a
```

Comment:
Can you move the text related to "diagVirtualTextHL" to below the text for "diagSignHintTexthl"?